### DISCUSSION TITLE
virtualenv support

### DISCUSSION THREAD

**Discussion by webbedfeet** [quarto-dev/quarto-cli#497] [Q&A] [answered]
Hello,

Is there a way to support virtualenv or conda envs in Quarto? Is it just a matter of activating the env and then running Quarto within it? 

Just thinking of the `use_virtualenv` use case from reticulate.

**Comment by jjallaire** [comment, marked as answer]
Correct, just activate the environment then call Quarto. See also https://quarto.org/docs/projects/virtual-environments.html

**Reply by webbedfeet** [reply]
Thanks. I missed the documentation. Nice job on the documentation by the way.

**Reply by kmcleste** [reply]
Did you have to do any additional configuration? I have my environment sourced and I'm able to use the quarto vscode extension to run interactive code chunks..but cannot actually render the full document

**Reply by peekxc** [reply]
I haven't always seen the result of `jupyter kernelspec list` aligned with the kernel listed in `quarto check jupyter`

**Reply by meteosimon** [reply]
I had the issue that `quarto check jupyter` did not identify the python of the activated conda env. I had to set the `QUARTO_PYTHON` variable in the `_environment` file.
